apps/server/src/modules/account/services/account.service.ts

```diff
@@ -129,6 +129,13 @@ export class AccountService extends AbstractAccountService {
 		await this.executeIdmMethod(async () => this.accountIdm.deleteByUserId(userId));
 	}
 
+	/**
+	 * @deprecated For migration purpose only
```

Comment:
Maybe add a TODO with a ticket-ID to better track when this instant legacy code will be removed (for every deprecated method)